find_xml_file.rb

```diff
@@ -0,0 +1,15 @@
+
+def find_xml_file(folder)
+  files = Dir.entries(folder)
+
+  xml_file = nil
+
+  files.each do |filename|
+    if filename.match "\.xml$"
+      xml_file = filename
+      break
+    end
+  end
+  
+  return xml_file
```

Comment:
what is xml_file is still nil (i.e., no match) - do you want to return an informative statement in this case?